install_files/ansible-base/roles/prepare-servers/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-# Ensures that prequisite packages for anssible and securedrop-admin install
+# Ensures that prerequisite packages for ansible and securedrop-admin install
```

Comment:
I'm pretty sure ansible here is a brand.

Sometimes `perquisite` may be intended, but I hope not.

Reply:
Fwiw, I call out `brand` occasionally for two reasons, one when internal case is important (JavaScript/GitHub/GitLab/...), but more typically when people have just misspelled it entirely which was the case here. -- It's a bit different from "this isn't in the dictionary".